megabytes.csv

```diff
@@ -0,0 +1 @@
+megabytes
```

Comment:
Yeah, I figured they're decent placeholders for when I eventually get tests in there.